src/content/docs/ai-monitoring/explore-ai-data/view-model-data.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: 'View AI model data in New Relic'
+metaDescription: 'AI monitoring lets you observe the AI-layer of your tech stack, giving you a holistic overview of the health and performance of your AI-powered app.'
+freshnessValidatedDate: never
+---
+
+import aiModelInventoryOverviewPage from 'images/ai_screenshot-full_model-inventory-overview-page.webp'
+
+import aiModelInventoryPerformancePage from 'images/ai_screenshot-full_model-inventory-performance-page.webp'
+
+import aiModelComparisonPage from 'images/ai_screenshot-full_ai-model-comparison-page.webp'
+
+With your AI app instrumented, you can explore data scoped to model performance and costs. While AI monitoring can give you insight into the performance of your AI app, it can also surface data specific to the models in your toolchain. 
```

Comment:
This intro paragraph is vague. What does "scoped to model performance and cost mean?" Can you add an action that contains a more direct action? I get that it improves costs, but how?